### PR TITLE
Flight Availability feature is added

### PR DESCRIPTION
**Pull request by annapoorni5**
#### Flight Availability feature is added ####

### CODE REVIEW

File: src/main/java/FlightAvailability.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Set;
+
+public class FlightAvailability {
+
+	static ArrayList<String> flightName = new ArrayList<String>();
```
Comment (csys-fresher-batch-2021) [inline src/main/java/FlightAvailability.java]: add private final

File: src/main/java/FlightAvailability.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	static {
+		System.out.println("********Select your flight name********");
+
+		flightName.add("IndiGo");
+		flightName.add("AirIndia");
+		flightName.add("SpiceJet");
+
+	}
+
+	public static void flightNameTest() {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/FlightAvailability.java]: Remove the unused method.

File: src/main/java/in/poorni/airlineapp/flightDetails.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
+package in.poorni.airlineapp;
+import java.util.ArrayList;
+
+public class flightDetails {
+	private static ArrayList<Flight> ListOfFlightDetailsList = new ArrayList<Flight>();
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/poorni/airlineapp/flightDetails.java]: Check naming convention for Classname, VariableNames.

File: src/main/java/in/poorni/airlineapp/flightDetails.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+	private static ArrayList<Source> ListOfSourceDetailsList = new ArrayList<Source>();
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/poorni/airlineapp/flightDetails.java]: Follow variable naming convention. Use List.

File: src/main/webapp/index.jsp [hunk 5]
```diff
@@ -10,11 +6,21 @@
...
 	<main class="container-fluid">
 		<h3>Welcome To Project</h3>
-		
-		<c:out value="Hello" />
-		<c:set var="now" value="<%=new Date()%>" />
-		<fmt:formatDate type="date" pattern="yyyy-MM-dd" value="${now}" />
+		<h3>Login to view flight details</h3>
+
 
+	<form>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/index.jsp:12]: create a separate login.jsp

File: src/test/java/in/poorni/airline/SourceDetailsTestCase.java [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+import org.junit.Test;
+
+import in.poorni.airlineapp.Source;
+import in.poorni.airlineapp.flightDetails;
+
+
+public class SourceDetailsTestCase {
+
+	@Test
+	public void testDispaly() {
```
Comment (csys-fresher-batch-2021) [inline src/test/java/in/poorni/airline/SourceDetailsTestCase.java]: Correct the spelling name for the method.

**Review by ghost — CHANGES_REQUESTED** [review]
Added Review comments

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**